docker/maintenance/cronjob-template.yaml

```diff
@@ -34,9 +35,11 @@ spec:
               limits:
                 cpu: "6"
                 memory: 1200Mi
+                ephemeral-storage: "200Gi"
               requests:
                 cpu: "2"
                 memory: 800Mi
+                ephemeral-storage: "200Gi"
```

Comment:
@r0bj is this size acceptable?

Reply:
It is a compromise. 80Gi limit for each part will give roughly 160Gi of storage needed + some spare in case of some weird content.

Reply:
I was thinking more about something around 50GB :). In general, having such a big ephemeral-storage request for your workload may cause trouble in scheduling workload, so your pod (created by this cronjob) may stay in `creating` state forever because it cannot fit to any node.
@harnash you mention 80GB for each part, let me understand it better:
Can we make those parts smaller (eg. 50GB)?
Can we process each part in sequence: create part, process part, upload part, whatever, go to next part? In this case we could make parts even smaller. 

So @themech we can go with 200GB but there is certain risk of not scheduling your pod. The bigger storage request the bigger the risk.

Reply:
@r0bj Yeah. We can make them smaller.